conbench/templates/compare-list.html

```diff
@@ -55,59 +55,44 @@ <h5 class="card-title"></h5>
-  is supposed to provide the more credible result.
-</p>
-{% if plot_history %}
-  <div class="col-md-12">
-    <h6>Top Outliers</h6>
-    <div id="bokeh-carousel"
-         style="visibility: hidden"
-         class="carousel slide"
-         data-ride="carousel">
-      <div class="carousel-inner" role="listbox">
-        <ol class="carousel-indicators">
-          {% if plot_history|length > 1 %}
-            {% for plot in plot_history %}
-              {% if loop.index == 1 %}
-                <li data-target="#bokeh-carousel" data-slide-to="0" class="active"></li>
-              {% else %}
-                <li data-target="#bokeh-carousel" data-slide-to="{{ loop.index - 1 }}"></li>
-              {% endif %}
-            {% endfor %}
-          {% endif %}
-        </ol>
-        {% for plot in plot_history %}
-          <div class="item  {% if loop.index == 1 %}active{% endif %} ">
-            <div align="center">
-              <a href="{{ outlier_urls[loop.index - 1] }}">{{ outlier_names[loop.index - 1] }}</a>
-            </div>
-            <div id="plot-history-{{ loop.index - 1 }}" align="center"></div>
-          </div>
-        {% endfor %}
-        {% if plot_history|length > 1 %}
-          <br>
-          <br>
-          <br>
-        {% endif %}
-      </div>
-    </div>
+<div class="collapse mb-3" id="collapseHelpblock">
+  <div class="card card-body">
+    <p>
+      Each row in the table below represents a comparison. The outcome of that comparison is summarized by two metrics: <code>change</code> and <code>z-score</code>.
+    </p>
+    <p>
+      There are no definite guidelines for interpretation. However, a
+      rather ignorant rule of thumb is that a <code>z-score</code> value of <code>-5</code> or smaller may indicate an actual performance regression
+      and should lead to brief human review.
+      A good method for such review is to click the link in the corresponding row; it leads to a visual representation of
+      the contender benchmark result, the baseline benchmark result, and the comparable historical ensemble.
+      That type of visualization usually allows a human to assess how significant a change really is.
+    </p>
+    <p>
+      The <code>change</code> column compares two matching benchmark results and expresses the difference in percent.
+    </p>
+    <p>
+      The <code>z-score</code> column shows how the contender benchmark result performs in view of a larger historical ensemble of comparable
+      benchmark results (in this case, the reference run contains only the newest of the results in that ensemble;
+      all other results were submitted via different CI runs in the past).
+    </p>
+    <p>
+      Whether the contender does better or worse than the reference is indicated by sign:
+      a negative sign (for both, <code>change</code> and <code>z-score</code>) reflects the direction of degradation, a positive sign reflects the direction of improvement.
```

Comment:
For follow-up, we could use to hit this harder visually. E.g. red/green text (or something more colorblind-friendly but still intuitive, if possible), or conditional formatting for the cell background, or something. 

Not sure exactly what the right solution is, but the goal is to minimize the time for a user to realize if a result is good or bad.

Reply:
Yes! There was something like that in place before. I'd love to revisit that when we have it easier to render the jinja template (not using the API json structures, but our more first-class typed objects).